vignettes/rls.Rmd

```diff
@@ -144,20 +131,25 @@ rls_check_status(con, "passwd")
 
 
 ``` r
-(policy1 <- rls_construct_policy(
-  name = "admin_all",
-  table = "passwd",
-  role = "admin",
-  using = "(true)",
-  check = "(true)"
-))
-#> <rls_policy>
-#>   policy name: admin_all
-#>   table: passwd
-#>   role: admin
-#>   using: (true)
-#>   check: (true)
-rls_create_policy(con, policy1)
+policy1 <- rls_tbl(con, "passwd") %>%
+  row_policy("admin_all") %>%
+  rows_existing(TRUE) %>%
+  rows_new(TRUE) %>%
+  to(admin)
```

Comment:
Where in the specification of this policy does the user specify that this policy is super permissive?

Reply:
Seems like if you don't specify any command you get all of them? I feel like I am missing something here.

Reply:
For the latter, yes `ALL` is the default https://www.postgresql.org/docs/current/sql-createpolicy.html - containing SELECT, INSERT, UPDATE, and DELETE.

Reply:
For the former,  from https://cord.co/techhub/architecture/articles/a-friendly-introduction-to-rls-policies-in-postgre#:~:text=Permissive%20policies%20are%20applied%20using,to%20be%20able%20to%20read. this is helpful

> Permissive policies are applied using a boolean “OR”, so you only need permission from one permissive policy to be able to query a certain row. Restrictive policies, on the other hand, are applied using a boolean “AND” — you have to pass all restrictive policies for every row you want to be able to read.

So by default all RLS policies are permissive. 

ALSO, moving from the work on main to this work I forgot about the permissive/restrictive bits. I'll add that back - the issue: https://github.com/getwilds/rls/issues/15